napari/components/camera.py

```diff
@@ -138,6 +138,25 @@ def set_view_direction(
         )
         self.angles = euler_angles
 
+    def look_at(
+        self, point: Union[Tuple[float, float], Tuple[float, float, float]]
+    ):
+        """Point the camera at a point in the 2D/3D scene.
+
+        In 2D, this method simply sets the camera center position. In 3D,
+        this keeps the camera center the same but updates the view direction,
+        ensuring that the point supplied as an argument is centered.
```

Comment:
This feels strange to me to look at, cause you lose track of where you were because of the rotation. It's also wird to interact with because I would expect the rotation center to be the point I'm looking at. So I would prefer the opposite (keep direction, change center). Is there a specific reason why you did it this way?

Reply:
the main motivation was convenience for an API we don't provide, changing the orientation so that we're looking at a specific point - this is the same behaviour as in three.js https://threejs.org/docs/#api/en/core/Object3D.lookAt 

I think part of the problem comes from our entangled camera position and center of rotation